### DISCUSSION TITLE
Unable to find entity

### DISCUSSION THREAD

**Discussion by umangr** [aahnik/tgcf#271] [Q&A]
Hi,
It was working perfectly till yesterday but now all of a sudden, it is unable to find groups via name (both public and private groups). Using the link still works. But if we use the name, it says entity not found (but for groups that I am the owner of, it's able to search via name)

**Comment by tissole** [comment]
For me is working. I run tests with chat name in source and chat ID in destination and with chat names in both source and destination. There were no issues.

**Reply by umangr** [reply]
Were you the administrators of said chats? Or were they public?

**Reply by tissole** [reply]
I was testing with 2 chats I created, so I was admin in both.
